recipe/bld.bat

```diff
@@ -1,4 +1,16 @@
-%PYTHON% build/build.py
+set BAZEL_VS="C:/Program Files (x86)/Microsoft Visual Studio/2019/BuildTools"
+set BAZEL_VC="C:/Program Files (x86)/Microsoft Visual Studio/2019/BuildTools/VC"
```

Comment:
```suggestion
set BAZEL_VS=%VSINSTALLDIR%
set BAZEL_VC=%VSINSTALLDIR%\VC
```
Not tested, but this may work too. With VSINSTALLDIR being set by the vc activation script.